e2e/Utils.js

```diff
@@ -160,11 +160,29 @@ export const extractSummary = (stdout: string) => {
   return {rest, summary};
 };
 
+const sortTests = (stdout: string) =>
+  stdout
+    .split('\n')
+    .reduce((tests, line, i) => {
+      if (['RUNS', 'PASS', 'FAIL'].includes(line.slice(0, 4))) {
+        tests.push([line.trim()]);
```

Comment:
I'd prefer to only right trim (or not trim at all) to line up the left side. That way thw column numbers and carets line up

Reply:
Great point!